folium/folium.py

```diff
@@ -440,6 +158,10 @@ def line(self, locations,
             or a tuple of the form (Vincent object, 'vis_path.json')
             It is possible to adjust the width of text/HTML popups
             using the optional keywords `popup_width` (default is 300px).
+        latlon: bool, default True
+            Whether locations are given in the form [[lat,lon]] or not ([[lon,lat]] if False).
+            Note that the default GeoJson format is latlon=False,
+            while Leaflet polyline's default is latlon=True.
```

Comment:
Yeah.  That was more a rant than a comment/review.  Just ignore me when I am talking about lon,lat ordering.